File: RC/Drivers/rc-ruby/fileManager.rb
class FileManager

	def initialize(file)
		@file = Dir.getwd + "/" + file
	end

	def read()
		handle = File.open(@file, "r")
		contents = handle.read
		handle.close
		return contents
	end

	def write(arr, noOfCmd)
		handle = File.open(@file, "w")
		i = 0
		while i < arr.size() && i < noOfCmd
			handle.write(arr[i])
			i += 1
		end
		handle.close
	end

	def makeFile()
		handle = File.open(@file, "w")
		handle.close
	end
end 

File: Bromine/RC/Drivers/rc-php/fileManager.php
<?php
  class FileManager{
  
    private $file;
    
    function __construct($file){
      $this->file = $file;
    }
    
    function read(){
      $handle = fopen($this->file, "r") or die("can't open file");
      $contents = fread($handle, filesize($this->file));
      fclose($handle);
      return $contents;
    }
    
    function write($arr){
      $handle = fopen($this->file, 'w') or die("can't open file");
      fwrite($handle, $arr);
      fclose($handle);
    }
    
    function makeFile(){
      $handle = fopen($this->file, 'w') or die("can't open file");
      fclose($handle);
    }   
  }
?>
